internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -601,8 +601,12 @@ func isSupportedPlatform(cluster *hivev1.ClusterDeployment) bool {
 	return true
 }
 
+// UserManagedNetworking is considered false if
+// 1. The cluster is an SNO (with no workers)
+// 2. The User specifically set this parameter to false
+// 3. Or, if the user did not set the value at all
```

Comment:
We should reflect this to users, as it affects the UX.
Please add this also to: https://github.com/openshift/assisted-service/blob/5d5320bfae959fc70b350d30144475f455f9381e/api/hiveextension/v1beta1/agentclusterinstall_types.go#L275-L277

Reply:
Fixed